plugins/available/python.plugin.bash

```diff
@@ -1,31 +1,31 @@
-cite about-plugin
+# shellcheck shell=bash
 about-plugin 'alias "shttp" to SimpleHTTPServer'
 
-if [[ "$OSTYPE" == 'linux'* ]]
-then
-  alias shttp='python2 -m SimpleHTTPServer'
+if _command_exists python2; then
+	alias shttp='python2 -m SimpleHTTPServer'
+elif _command_exists python; then
+	alias shttp='python -m SimpleHTTPServer'
```

Comment:
we can add the definition of the new http server in python3 (`python -m http.server`)

Reply:
This would be way better, but I don't use python much so I didn't want to make substantive changes. I'll make this change when I'm back at my computer. 👍